tests/test_make.py

```diff
@@ -2275,7 +2275,9 @@ class C:
             def __getstate__(self):
                 return ("hi",)
 
-        assert None is getattr(C(), "__setstate__", None)
+        assert getattr(object, "__setstate__", None) is getattr(
+            C, "__setstate__", None
+        )
```

Comment:
hm this makes no sense to me…this would pass if both don't exist, and if both are the same thing. this doesn't seem like a good test to me :)

maybe have two asserts, depending on version?

Reply:
But... that's what we want. If both don't exist, or both exist and are the same, the functionality is correct. Right?

Reply:
uh…i guess i'll take your word for it? it just seems like there's cases where it passes when it shouldn't 🤔

just merge the other shit in then ¯\_(ツ)_/¯